docs/docfiles/user_guides/manual/intro.rst

```diff
@@ -0,0 +1,34 @@
+.. _user_manual_intro:
+
+Introduction
+------------
+
+The Data Parallel Control (dpctl) package enables Python application developers
+and extension developers to access the data-parallel computing resource
+available on a modern heterogeneous system. Dpctl's objective is to provide
+a runtime to control or to manage data-parallel devices without the need for
+other Python packages to create such a runtime themselves. The runtime is built
+on top of the C++ SYCL standard and is envisioned to be both vendor and
+architecture agnostic. If the underlying SYCL runtime supports a type of
+architecture, the dpctl runtime will allow managing that architecture from
+Python.
+
+In its current implementation, dpctl supports only the DPC++ SYCL
```

Comment:
In its current form, `dpctl` relies on certain DPC++ extensions of SYCL standard. Moreover, the binary distribution of `dpctl` uses the proprietary Intel(R) oneAPI DPC++ runtime  bundled as part of oneAPI and supports Intel XPU devices only. However, `dpctl` is compatible with the runtime of open-source DPC++ sycl bundle that can be compiled to support a wide range of architectures (or backends?) including CUDA, AMD ROC, and HIP.

Reply:
Thanks!